src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/StringParsingHelpers.Statistics.cs

```diff
@@ -443,9 +444,9 @@ internal static IPInterfaceStatisticsTable ParseInterfaceStatisticsTableFromFile
             }
         }
 
-        private static uint ParseUInt64AndClampToUInt32(string value)
```

Comment:
Do we know why we were previously clamping to UInt32?  It seems like it was very intentional.

Reply:
I'm wondering if this was different in older kernel versions. I was thinking about it and using bigger type should work, right? e.g. if kernel uses int32 and we store it in long.

Reply:
@stephentoub ,
This is unclear.
This seems to be the original commit where this whole stuff was implemented:
https://github.com/dotnet/corefx/commit/d053fbd6f13a4da72ee7d604ad1eb82301452a43#diff-814c5ac438a49741588d3c0672d36374
And it was uint there in the beginning. Maybe @mellinoe can recall the reason?
This is the commit where parsing logic was changed to parse ulong and cut it to uint:
https://github.com/dotnet/corefx/commit/9176c6c4ddf837767b96cbb61c2d6a4690c0d974

Reply:
@wfurt , and it was always like this in some sence since public API surface was always using `long`
Original implementation https://github.com/dotnet/corefx/commit/d053fbd6f13a4da72ee7d604ad1eb82301452a43#diff-814c5ac438a49741588d3c0672d36374:
`  kernel value -> uint (without truncation) -> long`
Later fix https://github.com/dotnet/corefx/commit/9176c6c4ddf837767b96cbb61c2d6a4690c0d974: 
`  kernel value -> uint (with truncation) -> long`
This PR:
`  kernel value -> long (with truncation) -> long`